lib/mayaUsd/render/vp2RenderDelegate/proxyRenderDelegate.cpp

```diff
@@ -1770,6 +1796,19 @@ void ProxyRenderDelegate::ColorPrefsChanged()
     _RequestRefresh();
 }
 
+void ProxyRenderDelegate::ColorManagementRefresh()
```

Comment:
Can we have different color management settings for different viewports in multi-viewport mode? If so, how will it work?

Reply:
You can have different "view" transform for each viewport. It will take the resulting image, which was computed in the  "rendering" color space (sometimes called the "working" color space) and transform it into what is shown in the view.

So, my job is to bring the textures from their "source" color space into the "rendering" color space in order to compute the final image. I do not have to take into account any view dependent color spaces.